src/Microsoft.AspNetCore.Hosting.Abstractions/WebHostDefaults.cs

```diff
@@ -16,5 +16,7 @@ public static class WebHostDefaults
         public static readonly string ServerUrlsKey = "urls";
         public static readonly string ContentRootKey = "contentRoot";
         public static readonly string PreferHostingUrls = "preferHostingUrls";
+
+        public static readonly string ShutdownTimeoutKey = "shutdownTimeoutSeconds";
```

Comment:
@Tratcher Should we move this config out of Kestrel so there is one true place to set this?